src/toolkit/timeseries.cc

```diff
@@ -8,6 +8,15 @@ void RecordTimeSeries<POWER>(cyclus::Agent* agent, double value) {
   RecordTimeSeries<double>("Power", agent, value);
 }
 
+template <>
+void RecordTimeSeries<ENRICH_SWU>(cyclus::Agent* agent, double value) {
```

Comment:
It isn't meant to be archetype specific - it is just meant to indicate that the swu is associated with some sort of generalized material enrichment.  And this way - people know to associate the feed and swu series with each other.